manifests/config.pp

```diff
@@ -28,7 +18,7 @@
     group  => 'root',
   }
 
-  concat {  $strongswan::params::ipsec_conf:
```

Comment:
Without this change, declaring the main class with an override for `ipsec_conf` wouldn't do anything.

Reply:
It also now matches the value for `target` in the fragment below.